frontend/integration-tests/views/metalkube/dashboards.view.ts

```diff
@@ -0,0 +1,168 @@
+import { browser, element, by , $$ } from 'protractor';
+import { waitForNone } from '../../protractor.conf';
+
+export const untilNoLoadersPresent = waitForNone($$('.co-m-loader'));
+export const isLoaded = () => browser.wait(untilNoLoadersPresent).then(() => browser.sleep(2000));
+
+/**
+ * The label element in the nth row in the inventory card, which also says the total count of the
+ * items of the resource type (for example: "6 Nodes")
+ * @example
+ * await dashboardView.inventoryItemLabel(0).getText()
+ */
+export function inventoryItemLabel(rowNumber) {
+  return element(by.xpath(`(//div[@class="kubevirt-inventory__row-title"])[${rowNumber + 1}]`));
```

Comment:
I'm concerned about the brittleness of most of the selectors we're adding here. We should add in `data-test-id` elements for the exact element we want and should almost always avoid `xpath` in tests.

Here's an example of using a `data-test-id`:

https://github.com/openshift/console/blob/6579fb6fe3ebb55ab8016f3a844d8e82e09bf022/frontend/public/components/nav/nav-header.tsx#L47

https://github.com/openshift/console/blob/6579fb6fe3ebb55ab8016f3a844d8e82e09bf022/frontend/integration-tests/views/devconsole-view/dev-perspective.view.ts#L3